### ISSUE TITLE
Juglans regia genetic and QTL map

### ISSUE THREAD

**Issue by mestato** [statonlab/hardwoods_site#485]
First simple sequence repeat-based genetic linkage map reveals a major QTL for leafing time in walnut (Juglans regia L.)

http://scholar.google.com/scholar_url?url=https://link.springer.com/article/10.1007/s11295-019-1318-9&hl=en&sa=X&d=6113746859503588409&scisig=AAGBfm3Po1oAm1rxz9_koe8M6G8uMRHiHg&nossl=1&oi=scholaralrt&hist=UX7-rScAAAAJ:204383478123715844:AAGBfm0k2D5NRxX2fnjxDWoj-lEqBkIhQA

**Comment by almasaeed2010** [comment]
@jwest60 the file is located at https://static-content.springer.com/esm/art%3A10.1007%2Fs11295-019-1318-9/MediaObjects/11295_2019_1318_MOESM1_ESM.xlsx

Can you try to format it into a csv that the cmap_loader can load? 

Note that the position column has a `-` separating the start and stop so you are gonna need to split it.

**Comment by jwest60** [comment]
what should i put for the `feature_type_accession`? in the map for q rubra, it was

> For feature_type_accession field, markers starting with "loc" are SNPs, markers starting with anything else are "SSRs"

the names of the loci for this map seem to differ though and im not sure if the rule is the same here

**Comment by mestato** [comment]
They are all SSRs.

**Comment by jwest60** [comment]
thank you.

it also looks like some of the feature start and stop positions are not available

![image](https://user-images.githubusercontent.com/32902460/55652697-4bdfc300-57ba-11e9-98b7-598aaeefd81d.png)

how should these be handled?

**Comment by mestato** [comment]
The columns you want are "LG" and "Position in Consensus".

**Comment by jwest60** [comment]
finished with the file, here's the link to the sheet if anyone wants to see it

https://docs.google.com/spreadsheets/d/1wP83Zb__Y7B9OXqGuFqQy1z8T9MWOw5NLVVKHt2YtQ4/edit?usp=sharing

gonna try to add the genetic map to dev

**Comment by jwest60** [comment]
created the map and ran the cmap loader on the file, result was a vague sql error

![image](https://user-images.githubusercontent.com/32902460/55655307-d4ae2d00-57c1-11e9-9d7c-92e05b132f7c.png)

not 100% sure where the error is coming from, gonna try to work it out on my dev site

**Comment by jwest60** [comment]
looks like there are a few places in the map where the location was written with a `,` instead of a `.`

![image](https://user-images.githubusercontent.com/32902460/55656323-82bad680-57c4-11e9-9f41-dd46fd719d5e.png)

fixed the file, gonna try to run again

**Comment by mestato** [comment]
Oh yeah, Europeans often use commas where we use periods, it leads to a lot of confusion. I think you are safe to replace all the commas, nothing should be in the thousands of centimorgans.

**Comment by jwest60** [comment]
yeah but they only used commas for like 3 of the rows :thinking: 

successfully loaded on dev

![image](https://user-images.githubusercontent.com/32902460/55656808-412b2b00-57c6-11e9-82cd-22134ea8e5fc.png)

**Comment by jwest60** [comment]
[j_regia.cmap.gz](https://github.com/statonlab/hardwoods_site/files/3086265/j_regia.cmap.gz)
